_pages/events/courses.md

```diff
@@ -39,3 +39,25 @@ Glaciers are a common medium necessary to climb some of the bigger peaks in the
 {% template panel.html title: "Details" id: "glacier-detail-panel" %}
   {% include_relative courses/glacier.md %}
 {% endtemplate %}
+
+
+### Subsidies
+
+MITOC provides course subsidies to help offset the cost of learning outdoor safety skills.
+To be eleigible for a subsidy, one must lead (or co-lead) two MITOC trips within +/- 1 year of the course. The same trip cannot count for multiple subsidies. Subsidies are as follow.
+
+#### WFA: 
++ **Students**: Course is subsidized to cost **$100** (difference is reimbursed)
+
++ **Non-students**: Course is subsidized to cost **$150** (difference is reimbursed)
+
+#### All other safety courses (WFR, AIARE, Rock Rescue, Glacier Travel, etc):
++ All students recieve $100 off the full price of the course. Eligible for one per course.
```

Comment:
Did BOD vote to remove this subsidy for non-students? 

The notes from the meeting say BOD voted no and an increase, but I don't see anything saying we're removing it entirely for non-student?